RunForrest/dist/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html"; charset="UTF-8" />    
+<title>Diseño Web</title>
+<link href="main.css" rel="stylesheet" type="text/css">    
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.1.0/mustache.min.js"></script>
+<script type="text/javascript" src="js/main.js"></script>
+</head>
+<body class="page" onload="callPromise()">
+    <nav class="navbar">
+        <div class="navbar__logo">
+            <img src="images/logo.png" alt="logo">
+        </div>
+        <div class="navbar-menu">
+            <div class="navbar-menu__rectangle"></div>
+            <div class="navbar-menu__rectangle"></div>
+            <div class="navbar-menu__rectangle"></div>
+            <div class="navbar-menu__text">
+                <div class="linkAgents">Directorio de Agentes</div>
```

Comment:
Just as a matter of style, stick to one kind (Some classes use hyphen-case like `navbar-menu` and others camelCase like `linkAgents`). There is no 'correct style', but you should use one and be consistent with it. For example, using camelCase, underscore_case, hyphen-case, PascalCase, etc. It depends also in the language and other factors, but stick with one. Don't need to change it, but to take in mind :)